bin/tessel-2.js

```diff
@@ -19,9 +19,12 @@ const CLI_ENTRYPOINT = 'cli.entrypoint';
 
 // Check for updates
 const pkg = require('../package.json');
-updateNotifier({
-  pkg
-}).notify();
+
+try {
+  updateNotifier({ pkg }).notify();
+} catch (err) {
+  CrashReporter.submit(err.stack);
```

Comment:
It prints `INFO Crash Reported: http://crash-reporter.tessel.io/crashes?fingerprint=-0x173999f5` which is about as annoying as `WARN Failed to check for newer versions of the CLI`